camply/cli.py

```diff
@@ -91,32 +96,80 @@ def configure() -> None:
     "--provider",
     show_default=True,
     default="RecreationDotGov",
-    help="Camping Search Provider. Options available are 'Yellowstone' and "
-    "'RecreationDotGov'. Defaults to 'RecreationDotGov', not case-sensitive.",
+    help="Camping Search Provider. Options available are 'Yellowstone', "
+    "'RecreationDotGov', and 'GoingToCamp', Defaults to 'RecreationDotGov', Not case-sensitive.",
 )
 
 
+@provider_argument
+@rec_area_argument
+@camply_command_line.command()
+def equipment_types(
+    rec_area: Optional[int] = None, provider: Optional[str] = None
+) -> None:
+    """
+    Retrieve a list of equipment supported by the current provider/recreaton area
+
+    Equipment are pieces of equipment that can be used at a campsite. Different providers
+    and recreation areas have different types of equipment for which reservations can be made.
+    """
+    if not rec_area:
+        logger.error(
+            "This provider requires --rec-area to be specified when listing equipment types"
+        )
+        exit(1)
+
+    provider = provider.lower()
+    if provider == "goingtocamp":
+        GoingToCampProvider().list_equipment_types(rec_area[0])
+    else:
+        log_sorted_response(response_array=EquipmentOptions.__all_accepted_equipment__)
```

Comment:
Going to camp is the only provider with native equipment type support. Get rid of this else and exit cleanly.